doc/aliases/add_toggle.html

```diff
@@ -0,0 +1,14 @@
+<script type="text/javascript">
+/*
+  Creates a toggleable div that includes
+  the text/code associated to a button.
+*/
+var $getDiv = $('div.newInnerHTML').last();
+var buttonName = $getDiv.html();
+var label = getLabelName(buttonName.trim());
```

Comment:
For example, if written:
 @addtoggle{ C++} instead of @addtoggle{C++}

it would be seen as a different buttons, without that processing.
The idea here was to avoid those errors just by a silly mistake of an extra ' '.